### PR TITLE
Improve IE9 css

### PR DESCRIPTION
**Pull request by arothuis**
[Ticket 110246670](https://www.pivotaltracker.com/story/show/110246670)

### CODE REVIEW

**Comment by arothuis** [comment]
@rjkip @DRvanR

File: src/OpenConext/ProfileBundle/Resources/views/layout.html.twig [hunk 1]
```diff
@@ -9,14 +9,14 @@
     <link href="{{ asset('mdl/material.min.css') }}" rel='stylesheet' type='text/css' />
     <link href="{{ asset('css/font-awesome.min.css') }}" rel='stylesheet' type='text/css'>
     <link href="{{ asset('css/roboto.css') }}" rel='stylesheet' type='text/css'>
+    <!--[if lte IE 9]><link href="{{ asset('css/lte-ie9.css') }}" rel='stylesheet' type='text/css' /><![endif]-->
```
Comment (rjkip) [inline src/OpenConext/ProfileBundle/Resources/views/layout.html.twig]: I'd say either target IE using a selector (`.lte-ie9`) *or* using a separate stylesheet. Both seems unnecessary.

**Comment by DRvanR** [comment]
:+1:

**Comment by rjkip** [comment]
:+1:

**Pull request merged**